extensions/smallrye-openapi-common/deployment/src/main/java/io/quarkus/smallrye/openapi/common/deployment/SmallRyeOpenApiConfig.java

```diff
@@ -10,4 +13,11 @@
      */
     @ConfigItem(defaultValue = "/openapi")
     public String path;
+
+    /**
+     * If set, the generated OpenAPI schema documents will be stored here on build.
+     * Both openapi.json and openapi.yaml will be stored here if this is set.
+     */
+    @ConfigItem
+    public Optional<Path> storeSchemaDir;
```

Comment:
I would prefer if we used `Directory` rather than `Dir`.

Reply:
Cool. Changed to Directory